### ISSUE TITLE
Select element on the `/explore` page is missing an aria label

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7260]
### Where To Find Bug
`/explore`

### Expected Behaviour
The screen reader to announce the aria label given to the element.

### Actual Behaviour
The screen reader announces which option is selected and says "menu pop-up collapse item".
### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Click on the Explore tab
- Tab through the page until focus lands on the select item with "Trending" as the default option
- Listen to the screen reader say what is stated above for actual behavior

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)
